site/docs/releases.md

```diff
@@ -64,28 +64,36 @@ High-level features:
 
 * **Core**
   * Allow Iceberg schemas to specify one or more columns as row identifers [[\#2465](https://github.com/apache/iceberg/pull/2465)]. Note that this is a prerequisite for supporting upserts in Flink.
-  * Added JDBC [[\#1870](https://github.com/apache/iceberg/pull/1870)] and DynamoDB [[\#2688](https://github.com/apache/iceberg/pull/2688)] Iceberg catalog implementations.
+  * Added JDBC [[\#1870](https://github.com/apache/iceberg/pull/1870)] and DynamoDB [[\#2688](https://github.com/apache/iceberg/pull/2688)] catalog implementations.
   * Added predicate pushdown for partitions and files metadata tables [[\#2358](https://github.com/apache/iceberg/pull/2358), [\#2926](https://github.com/apache/iceberg/pull/2926)].
   * Added a new, more flexible compaction action for Spark that can support different strategies such as bin packing and sorting. [[\#2501](https://github.com/apache/iceberg/pull/2501), [\#2609](https://github.com/apache/iceberg/pull/2609)].
   * Added the ability to upgrade to v2 or create a v2 table using the table property format-version=2  [[\#2887](https://github.com/apache/iceberg/pull/2887)].
+* **Flink**
+  * Added support for SQL primary keys [[\#2410](https://github.com/apache/iceberg/pull/2410)].
+* **Hive**
+  * Added the ability to set the catalog at the table level in the Hive Metastore. This makes it possible to write queries that reference tables from multiple catalogs [[\#2129](https://github.com/apache/iceberg/pull/2129)].
+  * Deprecated the configuration property `iceberg.mr.catalog` which was previously used to configure the Iceberg catalog in MapReduce and Hive [[\#2565](https://github.com/apache/iceberg/pull/2565)].
+  * Enabled dropping HMS tables despite metadata problems [[\#2583](https://github.com/apache/iceberg/pull/2583)].
+  * Added table-level JVM lock on commits[[\#2547](https://github.com/apache/iceberg/pull/2547)].
```

Comment:
This is a bug fix so it should probably go in bugs. If possible, can you mention cases where one may have hit this issue?

Reply:
@rdblue are you referring to #2583?

Reply:
>  If possible, can you mention cases where one may have hit this issue?

@pvary and @marton-bod: Please provide the examples requested by @rdblue. Thanks.

Reply:
Nevermind then. Let's not block on this.

Reply:
The fix was following up on the issue: https://github.com/apache/iceberg/issues/2554. 
If the table becomes unloadable for any reason (e.g. the metadata files have been deleted/corrupted on the FS by some outside process), afterwards you were not able to ever drop your table from HMS before this fix

Reply:
Added. Done.